src/widgets/splits/ReplyInput.cpp

```diff
@@ -0,0 +1,58 @@
+#include "ReplyInput.hpp"
+
+#include "Application.hpp"
+#include "controllers/commands/CommandController.hpp"
+#include "controllers/hotkeys/HotkeyController.hpp"
+#include "messages/MessageThread.hpp"
+#include "widgets/helper/ResizingTextEdit.hpp"
+#include "widgets/splits/Split.hpp"
+
+namespace chatterino {
+
+ReplyInput::ReplyInput(QWidget *parent, Split *split)
```

Comment:
Should this really be a subclass? It already uses functionality from SplitInput, it has a Split parent, just like SplitInput. There already is logic for replies in SplitInput itself. Why is it a subclass in the first place?

Reply:
You're right, it doesn't really make sense now. I had initially subclassed the SplitInput to keep the logic separate but by adding inline replying at some point I introduced reply logic to SplitInput anyways.